draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.tsx

```diff
@@ -13,77 +15,88 @@ import {
   SecondaryActionsProps,
   TitleBlockMenuItemProps,
   convertSecondaryActionsToMenuItems,
+  DefaultActionProps,
 } from "./TitleBlockZen"
 import styles from "./MobileActions.module.scss"
 
-const buttonIsLink: (action: TitleBlockButtonProps) => boolean = action =>
+const menuItemIsLink: (item: TitleBlockMenuItemProps) => boolean = item =>
+  "href" in item || (!("component" in item) && typeof item.action === "string")
```

Comment:
Lol this is quite verbose

Reply:
Yeah it is a bit ott. I mainly whipped that up to not break the old deprecated "action" prop, which still has a bit of legacy code in there. We actually push peeps to use href and onclick in its place, which would remove the need to have such a gnarly check

Reply:
I'm leaning towards this being a breaking change for safety so potentially room to remove it entirely

Reply:
Breaking change might guarantee no one upgrades haha, keep it backwards compat. It's Almost Out The Door™